docs/administration/admin_panel/corporate_admin_panel.md

```diff
@@ -0,0 +1,13 @@
+---
+description: You can manage Corporate Accounts in the Admin Panel.
+---
+
+# Corporate
+
+You can manage Corporate Accounts in the Admin Panel.
+
+There, in the **Corporate** section, you can find find basic information about exisiting **Companies**, for example, Details, Versions, Locations, Translations, a list of members, billing addresses and technical details regarding the organization such as visibility, IDs, relations and so on.
```

Comment:
Here and later, please split paragraphs into shorter lines, ideally as sentence or clause end.